RHEL7 support

pierrecdn opened the pull request:
Hi Benjamin, 
We would like to propose our **RHEL7**-based support for this cookbook.
On our side, a wrapper cookbook on top of this one now allows us to deploy Aurora with a bunch of custom settings and customization.

Commit:
Add centos-7.1 tests

Commit:
Use mdsol Mesos cookbook instead of folsom-labs

msdol Mesos cookbook provides both support for Debian based
distributions as well as RHEL based.

Commit:
Refactor scheduler cookbook & attributes

Refactoring scheduler cookbook & attributes to be able to add support
for RHEL based distribs.

Commit:
Add Aurora scheduler support for Centos 7

Commit:
Refactor Aurora slave cookbook

This will allow to support for other distributions.

Commit:
Add Aurora slave support for CentOS 7

Commit:
Clean repositories setup

Commit:
Allow installing Aurora from a yum repository

Commit:
Rubocop compliance

Commit:
Fix aurora-scheduler package name

Commit:
Don't bind mesoslib to localhost by default

Commit:
Allow specifying Aurora version to install

Commit:
Fix typo in RHEL scheduler configuration file

Commit:
Add Mesos framework authentication support

Commit:
Fix Debian configuration when LIBPROCESS_IP is empty

Commit:
Document issue with replicated log initialization

pierrecdn committed:
Merge pull request #1 from criteo-forks/centos

Add support for RHEL based distributions

pierrecdn committed:
Ability to specify location of authentication file

benley commented:
wow, thanks for the pull request!  I should be able to review this sometime in the next couple of days.

Commit:
Make thermos-observer configurable on RHEL

Commit:
Minor refactoring

Commit:
Add client recipe to install Aurora tooling

pierrecdn committed:
Merge pull request #2 from criteo-forks/client

Add client recipe to install Aurora tooling

Commit:
Not installing Mesos repos on the client nodes

pierrecdn committed:
Allow using alternatives repository for mesos

By checking for mesos.repo attribute, we do the same thing done in the
mesos cookbook, allowing to use alternative repositories.

pierrecdn committed:
Merge pull request #3 from criteo-forks/alt_mesos_repo

Allow using alternatives repository for mesos

pierrecdn committed:
Mark sensitive file/template resources

Commit:
Implement configuration of Aurora basic authentication

Commit:
Merge pull request #4 from criteo-forks/policy

Implement configuration of Aurora basic authentication
